airbyte-integrations/connectors/source-github/source_github/schemas/commits.json

```diff
@@ -121,6 +121,47 @@
           }
         }
       }
+    },
+    "files": {
+      "type": ["null", "array"],
+      "items": {
+        "type": ["null","object"],
+        "properties": {
```

Comment:
@dlouseiro it's better to add the `additionalProperties: true` so in cases there is a missing or new property the connector won't break.

Reply:
Sure, makes sense! Will do